spec/default_req_handler_spec.rb

```diff
@@ -58,7 +58,8 @@
     expect(resp).to have_the_same_number_of_results_as(solr_resp_ids_from_query('k conway jill'))
   end
 
-  it 'history of the jews by paul johnson', jira: 'VUF-510' do
+  # mm setting at 8 makes this test fail
+  it 'history of the jews by paul johnson', jira: 'VUF-510', fixme: true do
```

Comment:
ditto to my comment above about explicit agreement to the tradeoffs.  This was an actual user search (b/c jira ticket number present)